FOCUS #93: Adding  availability zones dimension (#111)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/dimensions/availabilityzone.md

# Availability Zone

Availability Zones are physically separated and isolated areas within a region which provide high availability and fault tolerance.

> The AvailabilityZone column MUST be present in the billing data. This column MUST be of type String and MAY contain null values.

Reviewer: Should we make this column optional?
```suggestion
The AvailabilityZone column SHOULD be present in the billing data. This column MUST be of type String and MAY contain null values.
```

> Availability Zones are physically separated and isolated areas within a region which provide high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone transfer cost and usage based on where resources are deployed.

Reviewer 2: ```suggestion
Availability Zone is an identifier assigned to physically separated and isolated areas within a Region which provide high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone data transfer cost and usage based on where resources are deployed.
```

Author: thanks Udam, it has been updated.

## Column ID

AvailabilityZone

## Display name

Availability Zone

## Description

> Availability zones are physically separated and isolated areas within a region which provide high availability and fault tolerance.

Reviewer: ```suggestion
Physically separated and isolated areas within a region which provide high availability and fault tolerance.
```

Reviewer 3: It looks like this change was performed on the row 3 (within Definition) instead of the row 17 (within Description)

## Content constraints

| Constraint      | Value           |
|-----------------|-----------------|

> | Column required | True            |

Reviewer: If we make it optional...
```suggestion
| Column required | False           |
```

Author: Thanks Michael, just updated file after Squad 2 meeting

> Physically separated and isolated areas within a region which provide high availability and fault tolerance.

Reviewer 2: ```suggestion
A provider assigned identifier for a physically separated and isolated area within a Region that provides high availability and fault tolerance.
```
